sdk/storage/storage-blob/samples/typescript/withConnString.ts

```diff
@@ -0,0 +1,37 @@
+/* 
+ Setup: Enter your storage account name and shared key in main()
+*/
+
+import { BlobServiceClient } from "../../src"; // Change to "@azure/storage-blob" in your package
+
+async function main() {
+  // Create Blob Service Client from Connection String
+  const STORAGE_CONNECTION_STRING = "";
+  const blobServiceClient = BlobServiceClient.fromConnectionString(STORAGE_CONNECTION_STRING);
+
```

Comment:
Does connection string only work for Node.js? If so, (or cannot work with sharedkey), please add comment.

A customer recently complain about he wasted several hours then find out SharedKeyCredential is not supported in browsers...